backstory/src/main/java/com/google/sps/story/StoryManager.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.story;
+import java.io.IOException;
+
+/**
+ * Interface for story generation class.
+ */
+public interface StoryManager {
+
+  /**
+   * Returns generated text output for a given prompt, length, and temperature.
```

Comment:
Since the interface doesn't depend on prompt, length, and temperature, I'd recommend leaving them out here. For example, it's possible in the future some other implementation is added that uses a different model which doesn't have the concept of temperature.